qiskit_nature/properties/electronic_structure/integrals/two_body_electronic_integrals.py

```diff
@@ -0,0 +1,131 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The 2-body electronic integrals."""
+
+from typing import List, Optional, Tuple, Union
+
+import numpy as np
+
+from qiskit_nature import QiskitNatureError
+
+from .electronic_integrals import ElectronicIntegrals
+from ..bases import ElectronicBasis, ElectronicBasisTransform
+
+
+class TwoBodyElectronicIntegrals(ElectronicIntegrals):
+    """The 2-body electronic integrals."""
+
+    EINSUM_AO_TO_MO = "pqrs,pi,qj,rk,sl->ijkl"
+    EINSUM_CHEM_TO_PHYS = "ijkl->ljik"
+
+    ERI_TRUNCATION_LEVEL = 1e-12
```

Comment:
It might be useful to have quantity as a param on the constructor and default it to this value. That way if anyone wants they can build this out with a different threshold for truncation.